anchore_engine/clients/grype_wrapper.py

```diff
@@ -377,7 +377,10 @@ def init_grype_db_engine(
 
             # Store the dir and session variables globally
             # For use during reads and to remove in the next update
-            old_grype_db_dir = self._grype_db_dir
+            try:
+                old_grype_db_dir = self._grype_db_dir
```

Comment:
this was failing on initial sync, so i added this try except after consulting with @dspalmer99